src/hive/core.cljs

```diff
@@ -9,31 +9,48 @@
             [hive.queries :as queries]
             [hive.rework.util :as tool]
             [hive.components.screens.home.core :as home]
+            [hive.components.screens.errors :as errors]
             [hive.components.router :as router]
             [hive.components.screens.settings.core :as settings]
             [hive.components.screens.settings.city-picker :as city-picker]
             [cljs.core.async :as async]
             [cljs-react-navigation.reagent :as rn-nav]
-            [hive.components.screens.home.route :as route]))
+            [hive.components.screens.home.route :as route]
+            [hive.components.foreigns.react :as react]))
 
-"Each Screen will receive two props:
- - screenProps - Extra props passed down from the router (rarely used)
- - navigation  - The main navigation functions in a map as follows:
-   {:state     - routing state for this screen
-    :dispatch  - generic dispatch fn
-    :goBack    - pop's the current screen off the stack
-    :navigate  - most common way to navigate to the next screen
-    :setParams - used to change the params for the current screen}"
+(defn- screenify
+  [component title]
+  (let [id      (data/q queries/user-id (work/db))
+        data   @(work/pull! [{:user/route [:route/route :route/uuid]}]
+                            [:user/id id])]
+    (rn-nav/stack-screen 
+      (fn [props]
+        [:> react/View {:flex 1 :alignItems "stretch"}
+          (component props)
```

Comment:
@mehdisadeghi this should be `[component props]` See [this doc](https://github.com/reagent-project/reagent/blob/master/doc/UsingSquareBracketsInsteadOfParens.md) for more info.